index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Portfolio</title>
+	
+	<link rel="stylesheet" href="./styles.css">
+    </head>
+  <body>
+		<header class="nav">
```

Comment:
[OPTIONAL] Header on the web page has a large margin from the top. Please remove it.